jodconverter-docker/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM adoptopenjdk/openjdk8
+
+VOLUME /tmp
+VOLUME /libs
+
+ARG JODCONVERTER_WAR_NAME
+ADD ${JODCONVERTER_WAR_NAME} app.war
+
+ENV JAVA_OPTS=""
+
+RUN apt-get update && apt-get -y install vim wget curl unzip libice6 libsm-dev fontconfig libxt6 libxfont2 libcups2 libxrender1 gsfonts libreoffice \
```

Comment:
For build caching purposes, put the RUN before the ARG and ADD instructions

Reply:
do we need:
* both wget and curl ?
* vim ?

Reply:
I pushed a new commit, not sure why changes don't appear here ?